src/common/buffer.cc

```diff
@@ -2165,15 +2165,15 @@ std::ostream& buffer::operator<<(std::ostream& out, const buffer::ptr& bp) {
 }
 
 std::ostream& buffer::operator<<(std::ostream& out, const buffer::list& bl) {
-  out << "buffer::list(len=" << bl.length() << "," << std::endl;
+  out << "buffer::list(len=" << bl.length() << ",";
 
   for (const auto& node : bl.buffers()) {
     out << "\t" << node;
     if (&node != &bl.buffers().back()) {
-      out << "," << std::endl;
+      out << ",";
```

Comment:
```suggestion
      out << ",\n";
```

Reply:
@tchaikov 
Could we let the user to decide to add "\n" or not? When printing bufferlist with some other info (e.g. cout << buffer_list_instance << " wow" << endl;), it will occur new line ('\n') out of expected.

Reply:
"\n" here is the visual separator in between buffers in the bufferlist.

Reply:
Done